src/UnitySampleProject/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2022.3.26f1
```

Comment:
we shouldn't change this

Reply:
we should all be on .22f1 right?

Reply:
we updated to 2022.3.26 here https://github.com/ChainSafe/web3.unity/pull/938

Reply:
updating xD